shub/image/build.py

```diff
@@ -39,13 +39,15 @@
               help="stream build logs to console")
 @click.option("-V", "--version", help="release version")
 @click.option("-S", "--skip-tests", help="skip testing image", is_flag=True)
+@click.option("-n", "--no-cache", is_flag=True,
+              help="Do not use cache when building the image")
 @click.option("-f", "--file", "filename", default='Dockerfile',
               help="Name of the Dockerfile (Default is 'PATH/Dockerfile')")
-def cli(target, debug, verbose, version, skip_tests, filename):
-    build_cmd(target, version, skip_tests, filename=filename)
+def cli(target, debug, verbose, version, skip_tests, no_cache, filename):
+    build_cmd(target, version, skip_tests, no_cache, filename=filename)
 
 
-def build_cmd(target, version, skip_tests, filename='Dockerfile'):
+def build_cmd(target, version, skip_tests, no_cache, filename='Dockerfile'):
```

Comment:
There's another `upload` command which uses the `build_cmd` function, could you please update it accordingly? 
- https://github.com/scrapinghub/shub/blob/master/shub/image/upload.py#L48
- https://github.com/scrapinghub/shub/blob/master/tests/image/test_upload.py#L20

Reply:
Thank you for your reviews! @vshlapakov 

I tried to fix `upload` command.